plugins/chat/app/models/chat_message.rb

```diff
@@ -223,7 +223,7 @@ def full_url
   end
 
   def url
-    "/chat/message/#{self.id}"
+    "/chat/c/-/#{self.chat_channel_id}/#{self.id}"
```

Comment:
It's not really part of this PR, but I noticed we were not using the new message URL here